src/utils/Tmdb.js

```diff
@@ -0,0 +1,180 @@
+/**
+ * TMDB API client configuration
+ * @type {{key: string, baseUrl: string, imageBaseUrl: string}}
+ */
+const apiConfig = {
+  key: '5265606bb69e99437c85eb04dc6f29b5',
+  baseUrl: 'https://api.themoviedb.org/3/',
+  imageBaseUrl: 'http://image.tmdb.org/t/p/',
+};
+
+/**
+ * An abstraction layer above TMDB API https://developers.themoviedb.org/3/getting-started/introduction
+ */
+class _Tmdb {
```

Comment:
Why do you need `_` ?

Reply:
The module instantiates and exports a class object: `export const Tmdb = new _Tmdb();`
Added `_` to denote the class declaration as private and also to avoid identifiers conflict.
In overall, it's a singleton.
Will appreciate suggestions as to better pattern to employ.

Reply:
Thanks for pointing out at this and a hint re: `export default new...`. Fixed by ac1450e